### ISSUE TITLE
TypeError: CLIEngine is not a constructor with ESLint v8

### ISSUE THREAD

**Issue by EvgenyOrekhov** [NullVoxPopuli/eslint-plugin-decorator-position#457]
With ESLint 8.3.0 I get the following fatal error:

```
Failed to load plugin 'decorator-position' declared in '.eslintrc.json': CLIEngine is not a constructor
    at Object.<anonymous> (/example/node_modules/eslint-plugin-decorator-position/lib/rules/decorator-position.js:5:13)
```

**Comment by NullVoxPopuli** [comment]
yeah, I had started trying to support ESLint 8 (and ESLint 7) here: https://github.com/NullVoxPopuli/eslint-plugin-decorator-position/pull/441

but got distracted -- if you're motivated help would be extremely appreciated <3

**Comment by splincode** [comment]
Hello, any progress? We want to migrate from Eslint 8

**Comment by NullVoxPopuli** [comment]
nay, I haven't been real motivated

**Comment by raido** [comment]
I've hit this as well but I am good for now to stay with ESLint v7.

**Comment by NullVoxPopuli** [comment]
Should be resolved in v 5.0.0
